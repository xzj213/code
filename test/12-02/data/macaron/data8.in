10
0 3253 0 0 3253 0 3253 3253 0 3253
3253 3253 0 0 3253 3253 3253 0 3253 3253
0 3253 0 0 0 0 0 3253 3253 3253
3253 0 0 0 0 3253 3253 3253 3253 3253
3253 3253 0 0 0 3253 3253 3253 3253 0
3253 0 3253 0 0 3253 0 0 0 0
0 3253 3253 3253 0 3253 3253 3253 3253 3253
3253 0 0 0 3253 0 0 0 3253 3253
3253 3253 3253 3253 3665 3253 0 0 3253 3253
3253 0 3253 3253 3253 0 0 3253 0 950
