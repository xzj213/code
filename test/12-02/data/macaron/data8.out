0 338 740 950 3253 3559 3665 3843 
